recipes/libverto/all/test_package/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-find_package(libverto REQUIRED core CONFIG)
+find_package(libverto REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} PRIVATE libverto::verto)
+target_link_libraries(${PROJECT_NAME} PRIVATE libverto::liberto)
```

Comment:
oops I made a typo, it's libverto::libverto

Reply:
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE libverto::libverto)
```

Reply:
Hmm that's the first time I think I've seen a typo from you. Not bad